docs/content/en/docs/Developers/factorycube-edge/cameraconnect/index.md

```diff
@@ -65,7 +129,8 @@ Example value: 2021-0156
 
 ### TRIGGER
 
-**Description:** Defines the option of how the camera is triggered. Either via MQTT or via a continuous time trigger. <br>
+**Description:** Defines the option of how the camera is triggered. <br>
```

Comment:
Why do we have <br> tags here? they are not coming from you, but still weird

Reply:
i think they are relics from older formatting to ensure linebrakes in the tables, but i do not think they should be kept in the long term, as the exact linebreaks are devicedependent. 
Since this branch is already overloaded i did not want to touch them as well